src/ConvexHull.jl

```diff
@@ -106,3 +107,52 @@ function σ(d::AbstractVector{<:Real}, ch::ConvexHull)::AbstractVector{<:Real}
     ρ2 = dot(d, σ2)
     return ρ1 >= ρ2 ? σ1 : σ2
 end
+
+# ================================
+#  Convex hull of an array of sets
+# ================================
+"""
+    ConvexHullArray{N<:Real, S<:LazySet{N}} <: LazySet{N}
+
+Type that represents the symbolic convex hull of a finite number of convex sets.
+
+### Fields
+
+- `array` -- array of sets
+"""
+struct ConvexHullArray{N<:Real, S<:LazySet{N}} <: LazySet{N}
+    array::Vector{S}
+end
+# type-less convenience constructor
+ConvexHullArray(a::Vector{S}) where {S<:LazySet{N}} where {N<:Real} = ConvexHullArray{N, S}(a)
```

Comment:
Is this needed when there is no inner constructor? I remember that sometimes you need it...

Reply:
yes, its' needed, just checked